src/main/java/org/gridsuite/modification/server/WebFluxConfig.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.codec.ServerCodecConfigurer;
+import org.springframework.http.codec.json.Jackson2JsonDecoder;
+import org.springframework.http.codec.json.Jackson2JsonEncoder;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import org.springframework.web.reactive.config.WebFluxConfigurer;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@Configuration
+public class WebFluxConfig implements WebFluxConfigurer {
+
+    @Override
+    public void configureHttpMessageCodecs(ServerCodecConfigurer configurer) {
+        var objectMapper = objectMapper();
+        configurer.defaultCodecs().jackson2JsonEncoder(new Jackson2JsonEncoder(objectMapper));
+        configurer.defaultCodecs().jackson2JsonDecoder(new Jackson2JsonDecoder(objectMapper));
+    }
+
+    public static ObjectMapper createObjectMapper() {
+        var objectMapper = Jackson2ObjectMapperBuilder.json().build();
+        objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
why do we need this ? If I understand correctly, its to force jackson to serialize timestamps as datestrings instead of an integer. But we never have this in others microservices that deal with dates, what do we use instead? we should do the same everywhere (I think it's not very homogeneous already, between localdatetime and zoneddatetime... but adding timestamps is not going in the good direction I think)

Reply:
Removed